### PR TITLE
Build using AdoptOpenJDK 8 & 11

### PR DESCRIPTION
**Pull request by marcospereira**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,20 +1,44 @@
...
   include:
-    - stage: tests
-      script: sbt headerCheck test:headerCheck scalafmtCheck test:scalafmtCheck +test
     - stage: publish
       script: sbt +publish
+      env:
+        - TRAVIS_JDK=adopt@1.8.192-12
 
 stages:
-  - name: tests
```
Comment (marcospereira) [inline .travis.yml:27]: The tests will also run when publishing. Since they are pretty fast now, this should not have a significant impact.

Comment (ignasi35) [reply .travis.yml:27]: I'm confused. The `publish` stage only runs when the change was produced by a new tag

```
    if: repo = lagom/akka-discovery-service-locator AND tag =~ ^v
```

It seems to me like this PR makes the majority of builds a `noop`.

The idea of the condition for the stage `tests` (`NOT tag =~ ^v`) was exactly what you said: "Don't run the tests twice when publishing".

**Review by ignasi35 — CHANGES_REQUESTED** [review]
I think the AdopOpenJDK looks correct but the `tests` section removal is wrong.

**Review by ignasi35 — APPROVED** [review]

**Comment by ignasi35** [comment]
@marcospereira told me how/why this will actually work on a call.

**Comment by ignasi35** [comment]
> @marcospereira told me how/why this will actually work on a call.

Meaning: I'm good with the PR. We're not merging it yet because there's a couple of issues with the release @marcospereira is investigating and we don't want to add more variable into the mix. Once the release is 100% stable we'll come back to this PR

**Comment by ignasi35** [comment]
@marcospereira I think the releasing is fixed (`0.0.12`succeeded). So I'm good moving forward the merge of this.

**Comment by octonato** [comment]
Is that ok now?

**Comment by ignasi35** [comment]
> Is that ok now?

We want to expedite the resolution of the release process before merging this PR (which does quite a few changes in `.travis.yml`). I think it is OK now but wanted a last `+1` from @marcospereira

**Review by octonato — APPROVED** [review]

**Comment by octonato** [comment]
@marcospereira, it's on you. Feel free to merge.

**Pull request merged**